moderator.js

```diff
@@ -2,17 +2,19 @@ var lang = require('./languages');
 var bindings = require('./bindings');
 var secuence = require('./secuence');
 
-exports.getScriptFromDeck = function (deck) {
+exports.getScriptFromDeck = function (deck, selectedLanguage) {
```

Comment:
The idea with selectedLanguage it's for specifying the language, not the entire object.